packages/task/src/browser/task-service.ts

```diff
@@ -54,12 +57,28 @@ import { ProblemMatcherRegistry } from './task-problem-matcher-registry';
 import { TaskSchemaUpdater } from './task-schema-updater';
 import { TaskConfigurationManager } from './task-configuration-manager';
 import { PROBLEMS_WIDGET_ID, ProblemWidget } from '@theia/markers/lib/browser/problem/problem-widget';
+import { TaskNode } from './task-node';
 
 export interface QuickPickProblemMatcherItem {
     problemMatchers: NamedProblemMatcher[] | undefined;
     learnMore?: boolean;
 }
 
+interface TaskGraphNode {
+    taskConfiguration: TaskConfiguration;
+    node: TaskNode;
+}
+
+export enum TasksEndedTypes {
+    TaskExited,
+    BackgroundTaskEnded
+}
+
+export interface TaskEndedInfo {
+    taskEndedtype: TasksEndedTypes,
```

Comment:
Please rename `taskEndedtype` to `taskEndedType` following the [guidelines](https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#camelcase-var).

Reply:
fixed, done